docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
What is this file for? I haven't tested it since I am on Linux

Reply:
I have removed it. I copied this an other files (like `conf.py` and `Makefile`) from iMars3D but it really is not necessary to build the documentation.
Good catch! :mag: